CRM/Activity/Import/Parser/Activity.php

```diff
@@ -264,7 +264,7 @@ public function import($onDuplicate, &$values) {
     $session = CRM_Core_Session::singleton();
     $dateType = $session->get('dateTypes');
     if (!isset($params['source_contact_id'])) {
-      $params['source_contact_id'] = $session->get('userID');
+      $params['source_contact_id'] = CRM_Core_Session::singleton()->getLoggedInContactID();
```

Comment:
Is this effecient since $session is being called just above? I understand that you're doing things for consistency here, but wouldn't $session->getLoggedInContactID() be better here?